internal/services/signalr/signalr_service_resource.go

```diff
@@ -106,6 +111,46 @@ func resourceArmSignalRService() *pluginsdk.Resource {
 				},
 			},
 
+			"connectivity_logs_enabled": {
+				Type:     pluginsdk.TypeString,
+				Optional: true,
+				Computed: true,
+				ConflictsWith: []string{
+					"features",
+				},
+				ValidateFunc: validation.StringInSlice([]string{
+					"True",
+					"False",
+				}, false),
+			},
+
+			"messaging_logs_enabled": {
+				Type:     pluginsdk.TypeString,
+				Optional: true,
+				Computed: true,
+				ConflictsWith: []string{
+					"features",
+				},
+				ValidateFunc: validation.StringInSlice([]string{
+					"True",
+					"False",
+				}, false),
+			},
+
+			"service_mode": {
+				Type:     pluginsdk.TypeString,
+				Optional: true,
+				Computed: true,
+				ConflictsWith: []string{
+					"features",
+				},
+				ValidateFunc: validation.StringInSlice([]string{
+					"Serverless",
+					"Classic",
+					"Default",
+				}, false),
```

Comment:
is there a default value for this ("Default"?) - also this shouldn't be computed

Reply:
I've had to make these computed so the deprecation will function correctly, but I've left a note to remove for 3.0